images/proxy/Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM nginx:1.10.1
+FROM nginx:latest
```

Comment:
I haven't found something specific in nginx configuration which could be broken with the latest NGINX versions.